owncloudApp/src/main/java/com/owncloud/android/presentation/spaces/SpacesListFragment.kt

```diff
@@ -118,9 +129,25 @@ class SpacesListFragment : SpacesListAdapter.SpacesListAdapterListener, Fragment
 
     override fun onCreateOptionsMenu(menu: Menu, inflater: MenuInflater) {
         super.onCreateOptionsMenu(menu, inflater)
+        (menu.findItem(R.id.action_search).actionView as SearchView).run {
+            maxWidth = Int.MAX_VALUE
+            queryHint = resources.getString(R.string.actionbar_search)
+            setOnQueryTextListener(this@SpacesListFragment)
+        }
```

Comment:
Here we're setting a `OnQueryTextListener` to the element with ID `action_search`, and then in `FileDisplayActivity`, we're setting another similar listener to the element with ID `root_toolbar_search_view`. Those aren't the same, and maybe we just need one of them. This deserves a check to avoid unnecessary code 😄